Matey/Extensions/UIViewController + Ext.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIViewController + Ext.swift
+//  Matey
+//
+//  Created by Firat Polat on 30.08.2022.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController {
+
+    //alert controller
+    func showAlert(title:String?, message:String?, buttonName:String = "Done", completion: (()->())? = nil) {
```

Comment:
```suggestion
    func showAlert(title: String?, message: String?, buttonName: String = "Done", completion: (()->void)? = nil) {
```
It looks more readable this way.